components/gitpod-db/src/typeorm/webhook-event-db-impl.ts

```diff
@@ -51,4 +51,21 @@ export class WebhookEventDBImpl implements WebhookEventDB {
         query.limit(limit);
         return query.getMany();
     }
+
+    public async deleteOldEvents(ageInDays: number, limit?: number): Promise<void> {
+        const repo = await this.getRepo();
+        const d = new Date();
+        d.setDate(d.getDate() - ageInDays);
```

Comment:
:cloud: Would it make a difference to use `setUTCDate` here? :thinking:

Reply:
that line just applies a shift of days. as we alway initialize with date of server (you recall ;-)) it should be fine.